lib/siwapp/invoices.ex

```diff
@@ -1,54 +1,54 @@
-defmodule Siwapp.Invoices do
+defmodule Siwapp.Invoice do
```

Comment:
Same here, these files should be task of @RodrigoMolinaMadrid not yours. I'm not gonna merge nothing that's not your business.